h/browser/chrome/lib/hypothesis-chrome-extension.js

```diff
@@ -50,43 +51,33 @@ function HypothesisChromeExtension(dependencies) {
   });
   var tabErrors = new TabErrorCache();
 
+  restoreSavedTabState();
+
   /* Sets up the extension and binds event listeners. Requires a window
    * object to be passed so that it can listen for localStorage events.
    */
   this.listen = function (window) {
     chromeBrowserAction.onClicked.addListener(onBrowserActionClicked);
     chromeTabs.onCreated.addListener(onTabCreated);
+
+    // when a user navigates within an existing tab,
+    // onUpdated is fired in most cases
     chromeTabs.onUpdated.addListener(onTabUpdated);
```

Comment:
Think this comment would be good as a jsdoc comment on the `onTabUpdated()` function.